docs/running-on-mesos.md

```diff
@@ -151,13 +151,27 @@ Spark on Mesos also supports cluster mode, where the driver is launched in the c
 can find the results of the driver from the Mesos Web UI.
 
 To use cluster mode, you must start the MesosClusterDispatcher in your cluster via the `sbin/start-mesos-dispatcher.sh` script,
-passing in the Mesos master url (e.g: mesos://host:5050).
+passing in the Mesos master url (e.g: mesos://host:5050). This starts the MesosClusterDispatcher as an daemon running on the host.
+
+If you like to run the MesosClusterDispatcher with Marathon, you need to run the MesosClusterDispatcher in the foreground (i.e: `bin/spark-class org.apache.spark.deploy.mesos.MesosClusterDispatcher`).
```

Comment:
Nits: "an daemon" -> "a daemon", url -> URL, "i.e" -> "i.e.". You might back-tick MesosClusterDispatcher too.